spec/services/google/scrape_service_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::ScrapeService, type: :service do
+  describe '#call' do
+    context 'given a VALID keyword_id' do
+      context 'given NO errors' do
+        it 'saves the html of the result page', vcr: 'google/google_no_ads' do
+          keyword = Fabricate :keyword
+
+          described_class.new(keyword_id: keyword.id).call
+          updated = Keyword.includes(:keyword_search_entries).find(keyword.id)
```

Comment:
```suggestion
          updated_keyword = keyword.reload
```

Reply:
Fixed in 83904a70dcb01dce9120c1bd4bceb0109c3bb2c7